### DISCUSSION TITLE
pyarmor & pyinstaller both run successfully, but exe file cannot work

### DISCUSSION THREAD

**Discussion by TimJMM** [dashingsoft/pyarmor#1414] [Q&A]
pyarmor gen --enable-bcc -r @filelist.txt
it will encrypt 50 *.py files, run successfully. Then try python run, it works well.

pyinstaller --clean -n main -i favicon.ico main.py
pyinstaller  run successfully

BUT exe cannot work, please have a look

Error log:
dist\main>main.exe
Traceback (most recent call last):
  File "<frozen __main__>", line 5, in <module>
  File "<frozen main>", line 19, in <module>
ModuleNotFoundError: No module named 'XXX'
[7824] Failed to execute script 'main' due to unhandled exception!

Possible reason:
My pyarmor product name use Chinese characters, and version is professional, cause I want to use BCC & RFT
pyarmor -v
Pyarmor 8.2.9 (pro), 005261, 中文XXX

BTW for other try:
I am not try many times for RFT, For multiple files, it encrypt successfully but failed to python run. I am not sure why, BCC is enough for me.

I am try to build 4.3 step, but failed, I think it's due to Chinese characters in product name.
4.3. 详解可独立运行的加密脚本

**Comment by jondy** [comment]
如果加密后的脚步可以正常运行，请参考 `4.3 详解可独立运行的加密脚本` 和 PyInstaller 的文档解决这个问题。

对于复杂脚本，RFT 模式有可能需要进行一些人工配置，这需要你更加深入的了解 RFT 模式以及如何进行规则配置。

**Reply by 20007883** [reply]
请问你是看的哪个版本的pyinstaller文档呀，最新版的我找了下没有
